pyproject.toml

```diff
@@ -114,9 +110,10 @@ convention = "google"
 [tool.pytest.ini_options]
 norecursedirs = ["3rdparty"]
 addopts = ["--ignore=3rdparty"]
+markers = ["slow: marks tests as slow (deselect with '-m \"not slow\"')"]
```

Comment:
Nice, can you add a single example in our tests? Or does one exist?

Reply:
yeah this is a follow up to #589, I just forgot to add this config so pytest doesn't know to skip them 😢